dev/ci/ci-compcert.sh

```diff
@@ -12,8 +12,9 @@ if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 # CompCert does compile with -native-compiler yes
 # but with excessive memory requirements
 export COQCOPTS='-native-compiler no -w -undeclared-scope -w -omega-is-deprecated'
+if [ -z "${COQLIB}" ]; then COQLIB="${CI_INSTALL_DIR}/lib/coq"; fi
```

Comment:
```suggestion
```
ci-common sets COQLIB

Reply:
Apparently not in all cases https://github.com/coq/coq/blob/master/dev/ci/ci-common.sh#L35 ?

Reply:
That line says `export COQLIB=...`

Reply:
Maybe you mean it's not set in the other branch 

I don't think we want to set COQLIB there because we're interested in checking that the installed coq works without giving it hints 

Maybe we should set a (non exported) CI_COQLIB? IDK